internal/queue/queue.go

```diff
@@ -91,6 +92,24 @@ func (q *RabbitQueue) processPosts(conn db.Database) {
 	}
 }
 
+func (q *RabbitQueue) processComments(conn db.Database) {
+	postCQ := ConsumerQueue{
+		QueueName:  "comment_queue",
+		Exchange:   "federation",
+		RoutingKey: "comment.create",
+	}
+
+	aw := worker.CommentWorker{
+		Logger:     q.logger,
+		Repository: repository.NewRepository(conn),
+	}
+
+	err := q.StartConsumer(postCQ, &aw)
```

Comment:
Yeah... we need to rethink this. I don't think we want to create a separate consumer for everything. We probably just need one for actors and one for everything else. We can use the routing_key to pass the message to the appropriate function for processing.

Reply:
Sounds good will re-visit this tomorrow night and update.